src/main/java/org/codehaus/groovy/transform/stc/StaticTypeCheckingVisitor.java

```diff
@@ -29,6 +29,7 @@
 import groovy.transform.stc.ClosureParams;
 import groovy.transform.stc.ClosureSignatureConflictResolver;
 import groovy.transform.stc.ClosureSignatureHint;
+import jdk.internal.org.objectweb.asm.Opcodes;
```

Comment:
We should use `org.objectweb.asm.Opcodes` instead of `jdk.internal.org.objectweb.asm.Opcodes`

Reply:
oh, i see, thanks.